src/components/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -1,70 +1,73 @@
-          This step is not complete.
-          <router-link id="router-link" :to="{ path: '/add-people-roles', query: { showErrors: true } }">
-            Return to this step to complete it.
-          </router-link>
-        </span>
-      </div>
-    </div>
-
+  <div flat id="list-people-roles" class="mt-0">
     <!-- List Display Section -->
-    <div id="people-roles-list">
+    <div id="people-roles-list" v-if="getPeopleAndRoles.length > 0">
       <!-- List Headers -->
-      <v-row class="people-roles-header list-item__subtitle" no-gutters>
+      <v-row class="people-roles-list-header list-item__subtitle pb-3" no-gutters>
         <v-col v-for="(title, index) in tableHeaders" :key="index">
           <span>{{ title }}</span>
         </v-col>
-        <!-- Spacer Column For Actions -->
-        <v-col sm="1" v-if="!isSummary"></v-col>
+        <!-- Spacer Column for Actions -->
+        <v-col lg="1"></v-col>
       </v-row>
 
       <!-- List Content -->
+
+      <!-- FUTURE: either show the item or show the edit component -->
+      <!-- "if (edit_me) then ... else ..." -->
+      <!-- <template v-for="(orgPerson, index) in getPeopleAndRoles">
+        <v-row
+          class="people-roles-content list-item__subtitle"
+          :key="index"
+          no-gutters
+        >
+          <v-col>this row is to edit {{formatName(orgPerson)}}</v-col>
+        </v-row>
+      </template> -->
+
       <v-row
-        class="people-roles-content"
-        :class="{ 'list-item__subtitle': !isSummary }"
-        v-for="(officer, index) in personList"
+        class="people-roles-content list-item__subtitle py-3"
+        v-for="(orgPerson, index) in getPeopleAndRoles"
         :key="index"
-        no-gutters>
+        no-gutters
+      >
+        <!-- Name + Badge -->
         <v-col class="text-truncate">
-          <v-tooltip top :disabled="formatName(officer).length < 25" color="primary">
+          <!-- provide tooltip to display full name if name is longer than 25 chars -->
+          <v-tooltip top :disabled="formatName(orgPerson).length < 25" color="primary">
             <template v-slot:activator="{ on }">
-              <span v-on="on" class="people-roles-title"><strong>{{ formatName(officer) }}</strong></span>
+              <span v-on="on" class="people-roles-title">{{ formatName(orgPerson) }}</span>
             </template>
-            <span>{{ formatName(officer) }}</span>
+            <span>{{ formatName(orgPerson) }}</span>
           </v-tooltip>
+
+          <br v-if="orgPerson.action">
+          <v-chip v-if="orgPerson.action === ActionTypes.ADDED"
+            x-small label color="#1669BB" text-color="white">ADDED</v-chip>
+          <v-chip v-if="orgPerson.action === ActionTypes.EDITED"
+            x-small label color="#1669BB" text-color="white">CORRECTED</v-chip>
+          <v-chip v-if="orgPerson.action === ActionTypes.REMOVED"
+            x-small label color="#E0E0E0" text-color="grey darken-1">REMOVED</v-chip>
```

Comment:
Certainly doesn't have to be done here.. but perhaps we should create our own `v-chip` component. We have a lot of commonalities that could be passed as props, which could define the text/colours etc etc. 
Just thinking aloud here!

Reply:
Yup, we can do it in a future commit if it makes sense.